docker/Dockerfile

```diff
@@ -6,7 +6,7 @@ ENV NO_PROXY_CACHE="-o Acquire::BrokenProxy=true -o Acquire::http::No-Cache=true
 # Update and install dependencies without using any cache
 RUN apt-get update $NO_PROXY_CACHE  && \
   # $NO_PROXY_CACHE must not be used here or otherwise will trigger a hadolint error
```

Comment:
nit: we can remove the above comment as well.

Reply:
there was an actual bug in my changes. Fixed and removed a redundant `rm -rf`